packages/web/config/ibc-assets.ts

```diff
@@ -163,6 +163,44 @@ export const IBCAssetInfos: (IBCAsset & {
     coinMinimalDenom: "uscrt",
     isVerified: true,
   },
+  {
```

Comment:
Why are the tokens listed in ibc-assets twice each? Please remove these first five. Also I think these ics20 contract addresses are likely needed on the latter five.

Reply:
This looks the tokens to Secret Network. I modeled this PR after how the NETA token is defined since it's the same concept (cw20 token from Juno)

Reply:
yes, but they should not be listed twice each. Please remove the ones from line 166-203, but please add these ics20ContractAddresses to the listings starting at line 1071

Reply:
@JeremyParish69 this should be resolved.